Jasmine/Common/ViewModels/GridViewModel.swift

```diff
@@ -0,0 +1,129 @@
+import Foundation
+
+class GridViewModel: BaseViewModelProtocol {
+    weak var timeDelegate: TimeUpdateDelegate?
+    weak var scoreDelegate: ScoreUpdateDelegate?
+    weak var gameStatusDelegate: GameStatusUpdateDelegate?
+
+    /// Stores the grid data that will be used to display in the view controller.
+    var gridData: TextGrid
+    /// Number of rows in the grid, according to the answers property
+    var numRows: Int {
```

Comment:
remove since it's not needed in the protocol. go see what are the methods that are no longer declared in the protocol but it's still declared as non-private